zeppelin-web/src/app/notebook/notebook-actionBar.html

```diff
@@ -232,6 +232,7 @@ <h3>
                 class="btn btn-default btn-xs"
                 ng-click="removeNote(note.id)"
                 ng-hide="viewOnly"
+                ng-class="{'disabled':isNoteRunning()}"
```

Comment:
Maybe
```suggestion
                ng-class="{'disabled':isSequentialRun()}"
```
or am I missing something?

Reply:
isNoteRunning() Используется и в других кнопках в notebook-actionBar.html поэтому я решил не ставить здесь isSequentialRun(). С новым коммитом вообще уберу isSequentialRun из notebook.controller.js.